client/trigger.go

```diff
@@ -97,37 +128,6 @@ func TriggerThresholdOps() []TriggerThresholdOp {
 	}
 }
 
-// Allowed values for alert_type. | on_change is default
-const (
-	TriggerAlertTypeValueOnChange string = "on_change"
-	TriggerAlertTypeValueOnTrue   string = "on_true"
-)
-
-func (t *Trigger) MarshalJSON() ([]byte, error) {
```

Comment:
The 'less magic, more native Go' workflow of the Framework (vs the SDK) means this hack is no longer needed ✨